GETTING_STARTED.md

```diff
@@ -0,0 +1,121 @@
+Using this repo:
+
+This repo can be installed via cargo
+
+    cargo install clvm_tools_rs
+    
+The most current version of the language is in the rollup branch:
+
+    20221005-rollup
+    
+To install the current release:
+
+    cargo install clvm_tools_rs
+    
+To install from a specific branch:
+
+    
+    cargo install --no-default-features --git 'https://github.com/Chia-Network/clvm_tools_rs' --branch 20221005-rollup
+    
+To install a git checkout into your current python environment (must be in some kind of venv or conda environment):
+
+    git clone https://github.com/Chia-Network/clvm_tools_rs
+    cd clvm_tools_rs
+    maturin develop
+    
+Most people still compile chialisp via python.  One way to set up compilation
+in that way is like this:
```

Comment:
Why not just include this function in the wheel?

Reply:
Because it's not part of the classic interface.  Not a bad idea though to make a more batteries included thing.